### ISSUE TITLE
[QA]  Search by VAA ID for SOL doesn't work 

### ISSUE THREAD

**Issue by SamantaCasal** [XLabs/wormscan-ui#142]
**Environment:**
MacBook Pro, macOS Ventura 13.1.1, Chrome Version 113.0.5672.63.
**Prerequisite:** In the test the ID was used: 1/ec7372995d5cc8732397fb0ad35c0121e0eaa90d26f828a534cab54391b3a4f5/298270

**Steps**
1. https://front.staging.wormscan.io/#/
2. Enter the SOL ID
3. Open network in console(in dev tools)
4. Click on the search icon
5. Oberve the response on the network

**Obtained result:**
In the response you can see that the call https://api.staging.wormscan.io/api/v1/vaas/1/ec7372995d5cc8732397fb0ad35c0121e0eaa90d26f828a534cab54391b3a4f5/298270?page=0&pageSize=10&sortOrder=ASC is successful and has data to display, however, the data is not displayed and instead you see the not found message.
**Note:** The txHash endpoint throws an error, however, the search is done by ID, therefore the data is expected to be displayed

**Expected result:**
You should see the detail of the transaction

**Evidence:**

https://github.com/XLabs/wormscan-ui/assets/69044575/f28a701b-aa64-430f-9aaa-f11878ce8628

![Screenshot 2023-06-22 at 12 53 14 AM](https://github.com/XLabs/wormscan-ui/assets/69044575/1e9ccb2c-3066-4c9e-ac65-a0c9da6b72ff)

**Comment by agodnic** [comment]
I believe this should be fixed for the case of Solana by https://github.com/wormhole-foundation/wormhole-explorer/commit/532412077b6a5b6ff02850ffc6bd3005b339da15.

Aptos is not ready yet, I'm still working on it.

**Comment by SamantaCasal** [comment]
@raop155 , if you are looking for the VAA ID, it should show the data that comes from the /api/v1/vaas/chainID/emitter_address/sequence endpoint. That was my point, maybe I'm wrong

**Comment by raop155** [comment]
@SamantaCasal I can do that also as second option, but as @agodnic says the txHash should be fixed tho :)
I don't want to depend on two queries on the `tx detail page` if it is not needed for now.

**Comment by agodnic** [comment]
FWIW looking up a VAA by `chain/emitter/seq` is more robust than looking it up by `txHash`.

Rationale: the `chain/emitter/seq` tuple has immediate consistency and always returns the VAA. On the other hand, the `txHash` has eventual consistency and is prone to internal errors due to backend components failing, which means that sometimes you won't find the VAA you're looking for.

**Comment by SamantaCasal** [comment]
If you don't want to make 2 calls, wouldn't it make more sense to make only the one that is by VAA Id since the search is by VAA Id?

**Comment by agodnic** [comment]
> If you don't want to make 2 calls, wouldn't it make more sense to make only the one that is by VAA Id since the search is by VAA Id?

I agree with this.

The URL would be `https://front.staging.wormscan.io/#/tx/{emitterChain}/{emitterAddress}/{sequence}` instead of `https://front.staging.wormscan.io/#/tx/{txHash}`.

This is going to be way more robust due to the eventual consistency problems that the `txHash` field has.

**Comment by raop155** [comment]
I can do that, but will have the same problem with the `txHash` displayed in the `Tx detail`.
So it is not a `full` solution ? hmm 🤔 
I think we can wait till we have proper txHashes

**Comment by agodnic** [comment]
@raop155 even when the `txHash` field is fixed for Solana and Aptos, the eventual consistency problem still exists.

It will be more robust to identify VAAs based on their `chain/emitter/seq` (which we always have) instead of their `txHash` (which sometimes will have delay, and sometimes we won't have due to internal errors).

**Comment by raop155** [comment]
Ok I will handle this in other way then. Ill pass this to TODO

**Comment by SamantaCasal** [comment]
QA OK! 

https://api.staging.wormscan.io/api/v1/vaas/1/ec7372995d5cc8732397fb0ad35c0121e0eaa90d26f828a534cab54391b3a4f5/298270?parsedPayload=true&page=0&pageSize=10&sortOrder=ASC
